nuxt-app/pages/components/notification.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
please can you add this to the SSR tests? 

*Refs:*
- https://github.com/justeattakeaway/pie/wiki/SSR-Testing
- https://github.com/justeattakeaway/pie-aperture/blob/main/test/ssr/ssr.spec.ts